assets/airports/ksea.json

```diff
@@ -215,11 +293,17 @@
                 "34C": ["NESOE"],
                 "34R": ["DODVE"]
             },
-            "body": [["WUREL", "A31+|S250-"], ["ATOME", "S250-"], ["CENEX", "S250-"], ["KTSAP", "S250-"], ["ZUVEN", "A100+|S250-"], ["KMORE", "S250-"]],
+            "body": [["WUREL", "A31+"], "ATOME", "CENEX", "KTSAP", ["ZUVEN", "A100+"], ["KMORE", "S250-"]],
             "exitPoints": {
-                "BLUIT": ["BLUIT"]
+                "BLUIT": ["BLUIT"],
+                "KMORE": []
             },
-            "draw": [["CUSBU", "WUREL"], ["NESOE", "WUREL"], ["DODVE", "WUREL"], ["WUREL", "ATOME", "CENEX", "KTSAP", "ZUVEN", "KMORE", "BLUIT*"]]
+            "draw": [
+                ["CUSBU", "WUREL"],
+                ["NESOE", "WUREL"],
+                ["DODVE", "WUREL"],
+                ["WUREL", "ATOME", "CENEX", "KTSAP", "ZUVEN", "KMORE*", "BLUIT*"]
+            ]
         },
         "KTSAP5": {
```

Comment:
Must have been removed the other day when they added the ATOME1. Cool!

Removed.